app/controllers/v1/sessions_controller.rb

```diff
@@ -55,6 +55,10 @@ def saml_callback
         redirect_to url_service.login_redirect_url(auth: 'fail', code: auth_error_code(saml_response.error_code))
         callback_stats(:failure, saml_response, saml_response.error_instrumentation_code)
       end
+    rescue SAML::UserAttributeError => e
+      log_message_to_sentry(e.message, {}, {}, :warning)
+      redirect_to url_service.login_redirect_url(auth: 'fail', code: e.code)
```

Comment:
I think you want `log_message_to_sentry(e.message, :warning, {}, {})`

Apparently the method signature for log_message and log_exception are different :/

Reply:
We also might want to add a tag here. Checking w/ our sentry team

Reply:
Good catch, thank you. Fixed in latest commit.

Reply:
Awesome. I'll be working to fix those methods because that kind of surprise shouldn't happen.